src/Cache/MomentoServiceProvider.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Momento\Cache;
+
+use Illuminate\Support\ServiceProvider;
+
+class MomentoServiceProvider extends ServiceProvider
```

Comment:
can you please create a ticket on the dev eco board to remind us to try to PR this directly into laravel once we have added the official support for increment/decrement/putMany in the future?